### PR TITLE
Add unneededtclist update

### PR DESCRIPTION
**Pull request by mcd500**
Adding an unneeded-tc-list field in the example of Update message in Appendix D.
Based on the comment at https://github.com/ietf-teep/teep-protocol/pull/97

@dthaler please review, thanks

### CODE REVIEW

File: draft-ietf-teep-protocol.md [hunk 1]
```diff
@@ -1260,23 +1262,23 @@ token = 20
...
+    1A 77777777          # unsigned(2004318071, 0x77777777)
+    05                   # unsigned(5)
+    01                   # unsigned(1) within .size 4
+    06                   # unsigned(6)
+    00                   # unsigned(0) within .size 4
+    08                   # unsigned(8)
+    82                   # array(2)
+      81                 # array(1)
+        4F               # bytes(15)
+          0102030405060708090A0B0C0D0D0F
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:1278]: A bunch of places have ...0D0D0F instead of 0D0E0F.  Is that intentional?

**Review by dthaler — APPROVED** [review]
Looks good except for one question about whether there's a typo in several places.

**Comment by dthaler** [comment]
Does this PR obsolete #97?  if so, can we close 97?

File: draft-ietf-teep-protocol.md (hunk 1 above)
Comment (mcd500) [reply draft-ietf-teep-protocol.md:1278]: Thanks for catching them. All 0D0D0F are typo of 0D0E0F, I will revise the PR.

**Comment by mcd500** [comment]
I will rebase this PR on top of https://github.com/ietf-teep/teep-protocol/pull/97 to reduce the changes in this PR

File: draft-ietf-teep-protocol.md (hunk 1 above)
Comment (mcd500) [reply draft-ietf-teep-protocol.md]: I added a commit of fixing all 0D0D0F to 0D0E0F in this PR

**Pull request merged**